pmmdemo/provision_scripts/bastion.yml

```diff
@@ -29,6 +29,10 @@ runcmd:
   - docker exec nginx apt update
   - docker exec nginx apt install -y certbot python3-certbot-nginx
   - docker exec nginx certbot --nginx --non-interactive --agree-tos -m ${email} -d ${domain}
+  - echo "${pmm_admin_pass}" > /root/admin_password
+  - chmod 400 /root/pmm-admin-password
+  - timeout 120 bash -c 'until curl --request 'GET' --insecure https://${pmm_server_endpoint}/v1/readyz ; do sleep 3; done'
```

Comment:
shouldn't this be using consul? i thought that was the idea of having it to know when services are ready?

Reply:
solved in #114 :guitar: